sklearn/preprocessing/data.py

```diff
@@ -2214,9 +2212,12 @@ def _transform_col(self, X_col, quantiles, inverse):
             upper_bound_x = 1
             lower_bound_y = quantiles[0]
             upper_bound_y = quantiles[-1]
-            #  for inverse transform, match a uniform PDF
+            #  for inverse transform, match a uniform distribution
             with np.errstate(invalid='ignore'):  # hide NaN comparison warnings
-                X_col = output_distribution.cdf(X_col)
+                if output_distribution == 'normal':
```

Comment:
how about we use `!= 'uniform'` to make this a bit more future-ready.

Reply:
and still use getattr(stats, output_distribution) as well?

Reply:
Yeah, okay... I'm not too fussed.